java-grupa8/homework-1/tema/src/main/java8/ro/siit/tema/BubbleSort.java

```diff
@@ -25,5 +25,11 @@ public static void bubbleSort(SalesRepresentatives[] SalesGuy) {
         }
         while (swap > 0);
 
+        for (int i = 0; i < SalesGuy.length; i++) {    //printing on the screen the results of the bubble sort
```

Comment:
This code is part of another task(homework). It should not show up in this pull request.